src/mixins/filing-template-mixin.ts

```diff
@@ -483,74 +483,71 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
    * @param entitySnapshot the latest entity snapshot
    */
   parseCorrectionFiling (filing: CorrectionFilingIF, entitySnapshot: EntitySnapshotIF = null): void {
-    // make a copy so snapshot objects are different from objects below
-    entitySnapshot = cloneDeep(entitySnapshot)
```

Comment:
This doesn't solve anything because parts of the new entitySnapshot can still be assigned (by reference) to different variables below and cause the snapshot object to be updated accidentally when we only mean to update the store working data.